app/views/merchants/invoices/show.html.erb

```diff
@@ -3,4 +3,13 @@
   <p> Invoice Status: <%= @invoice.status  %></p>
   <p> Created on: <%= @invoice.created_at.strftime("%A, %B %d, %Y")%></p>
   <p> Customer Name: <%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %></p>
+</div>
+
+<div id= "item_info">
+  <% @merchant.items_for_this_invoice(@invoice.id).each do |invoice_item| %>
```

Comment:
Just a thought for you: this `items_for_this_invoice` method is sort of duplicating what the invoice_item table's associations are already doing, ie: you really don't need a new method to get this data.
For example, you could just base your iteration out of the invoice, and change line 9 of your view to this: `@invoice.invoice_items.each do |invoice_item|`